### ISSUE TITLE
Additional files in migrations folder

### ISSUE THREAD

**Issue by saveman71** [tj/node-migrate#145]
I'm currently "migrating" (🥁) our migrate installation from 0.1.6 to the latest. I have some helpers files (e.g. `migrations/helpers.js` that is required in almost every migration I create.

However it seems it's trying to use `helpers.js` as a migration file now (`Migration helpers.js does not have method up`), whereas this file was ignored before.

It seems this use case is confirmed to be possible in the README:

https://github.com/tj/node-migrate/blob/ee1df149484f342fa693c610d6bb708214d59729/Readme.md#L84-L88

What's the best way of fixing that?

**Comment by saveman71** [comment]
I agree moving them out seems like the better option. However I still think the doc (snippet I ref'd in https://github.com/tj/node-migrate/issues/145#issue-403236401) is a bit misleading on that, as it seems it still should be possible without any further configuration.

Also, the breaking change wasn't mentioned in [History.md](https://github.com/tj/node-migrate/blob/ee1df149484f342fa693c610d6bb708214d59729/History.md) :)

(no reproach at all btw, congrats on maintaining this properly already and answering this that fast :clap:)

I might make a PR later in the week for these documentation fixes. If I don't do it feel free to go ahead!

**Comment by wesleytodd** [comment]
Hey @saveman71, no ill will taken!  I changed so much in the 1.0 I am sure I missed some things in the history, which is why OSS and team work is great!  A PR will be very welcome, but please be patient with me now because between my comment 6 days ago and yours 3 days ago I had a baby daughter (👶!!!) so will be a bit disconnected.

If you make a PR for the docs it would be great to also update the history file.  Thanks!

**Comment by gustavopch** [comment]
If I understand correctly the issue, the desired feature already exists in the CLI as an option called `matches`.
```
migrate up --matches '^\d+-[\w-]+.js$'
```
See: https://github.com/tj/node-migrate/blob/master/bin/migrate-up#L25

**Comment by mkovel** [comment]
@gustavopch  Unfortunately --matches  logic was implemented base on **minimatch** and do not support full Regexp only glob and this  example does not work
If i wrong, please let me know.

**Comment by monooso** [comment]
@gustavopch / @mkovel The `--matches` flag will suffice if you place all of your "helper" files in a sub-directory. For example:

```
migrate up --matches '!helpers'
```

**Comment by mkovel** [comment]
> @gustavopch / @mkovel The `--matches` flag will suffice if you place all of your "helper" files in a sub-directory. For example:
> 
> ```
> migrate up --matches '!helpers'
> ```

good point. tnx
